index.js

```diff
@@ -3,32 +3,41 @@ function switchDebug(){
     debug = !debug;
 }
 
-function randomNumberBetween(min, max) {
-    min = Math.ceil(min)
-    max = Math.floor(max)
-    var r = Math.random()
-    return Math.floor(r * (max - min + 1) + min)
-}
+function selectRandomArrayItem(input, options) {
```

Comment:
could make options a global and then use the options input as a way to update the options.

then make the options an object 
```
{
    inculdeIndex: false,
    returnRange : 1
}

Reply:
then that also would make it easier to extend later if needed

Reply:
this new commit should solve the problem. refer https://github.com/TechDevils/array-random-picker/pull/3/commits/1d52e8ad0a831115a2774b39ac86e007c577b4a2